app/src/main/res/values/strings.xml

```diff
@@ -287,7 +289,8 @@
     <string name="notification_favourite_description">Notifications when your toots get marked as favorite</string>
     <string name="notification_poll_name">Polls</string>
     <string name="notification_poll_description">Notifications about polls that have ended</string>
-
+    <string name="notification_subscription_name">Subscriptions</string>
```

Comment:
Subscriptions is the better wording, but maybe we should match Mastodon web for consistency? ("New toots")

Reply:
I replaced it with "New toots" and replaced posts by toots in description.